.bazelrc

```diff
@@ -0,0 +1,3 @@
+build --macos_minimum_os=12.0
+build --repo_env=CC=clang
+test --test_output=errors
```

Comment:
Should there be some configurations here to allow for debug and release builds?

Reply:
Which kind of configurations are you thinking about adding?

Reply:
I would imagine that would be up to the end consumer of the binary, they could specify `Bazelisk build @SourceKitten//:sourcekitten -c dbg` or '@SourceKitten//:sourcekitten -c opt'. Unlike SPM Bazel is decentralizing this configuration to the end user, vs hosting all config on this library

Reply:
Great, didn't know those configurations were available by default 👍